### PR TITLE
Use proper capistrano hook depending on capistrano version

### LINKED ISSUE #324: Generator should generate correct capistrano integration
At the moment, the generator probably assumes capistrano v2 and always adds this integration. The generator should check the capistrano version and install the correct integration.

for cap v2:
```ruby
require './config/boot'
require 'airbrake/capistrano'
```

for cap v3:
```ruby
require 'airbrake/capistrano3'
after :"deploy:restart", "airbrake:deploy"
```

### PR DESCRIPTION
**Pull request by shifi**
Fixes #324

### CODE REVIEW

**Comment by dvdplm** [comment]
Are we sure we even want to rewrite peoples deploy scripts like this? Is this common practise for generators? :/

**Comment by vmihailenco** [comment]
Not sure I follow, but why not? Looks like we use capistrano hook to notify Airbrake about deploy. Looks okay.

**Comment by shifi** [comment]
We just append capistrano hook at the end of deploy script. We did it like
that before, but we always used hook for capistrano version 2

File: lib/rails/generators/airbrake/airbrake_generator.rb [hunk 1]
```diff
@@ -47,18 +47,42 @@ def ensure_plugin_is_not_present
...
+    !Gem.loaded_specs['capistrano'].nil? &&
+      File.exists?('config/deploy.rb') &&
+      File.exists?('Capfile')
+  end
 
-        require './config/boot'
-        require 'airbrake/capistrano'
-      HOOK
+  def append_capistrano_hook
+    if capistrano_version < Gem::Version.new('3')
```
Comment (phumpal) [inline lib/rails/generators/airbrake/airbrake_generator.rb]: cap3 is now 2 years old. How long should we be concerned with supporting v2?

**Comment by phumpal** [comment]
I tend to agree with @dvdplm, however, in this instance we improve usability so LGTM.

**Pull request merged**